site.region

```diff
@@ -30,9 +30,86 @@
     <noscript>
       <img src="https://secure.leadforensics.com/77932.png" style="display: none;" />
     </noscript>
-  </head>
-  <body>
+    <script>
+      var colourOnButtonText = 'Make my day accessible';
+      var colourOffButtonText = 'Switch back to colour';
+
+      function initialiseAccessibilityBar() {
+        document.getElementById('accessibilityDayButton').style.display = 'block';
+      }
+
+      function toggleGreyscale() {
+        document.documentElement.classList.toggle('accessibilityDay');
+
+        var button = document.getElementById('accessibilityDayButton');
+        if (button.innerHTML == colourOffButtonText) {
+          button.innerHTML = colourOnButtonText;
+        } else {
+          button.innerHTML = colourOffButtonText;
+        }
+      }
+    </script>
+    <style>
+      .accessibilityDay {
+        filter: grayscale(100%);
```

Comment:
As with the comment on the Website Honestly PR, we just need to include the following filters in addition to what's there already:

    -webkit-filter: grayscale(100%);
    -moz-filter: grayscale(100%);
    -ms-filter: grayscale(100%);
    -o-filter: grayscale(100%);